### PR TITLE
[DNM] Fixes every sec balancing problem, gives them rocket launchers in the armory.

### PR DESCRIPTION
**Pull request by Paxilmaniac**
## About The Pull Request
Does EXACTLY as the title says, makes a new rocket launcher subtype with a new look and mindshield firing pin, and then makes a whole roster of new rockets for them to fire at criminals (definitely criminals).

**Smoke Rocket:**
 Does exactly what you think it would, makes a big cloud of smoke and hurts to get hit directly with.

**White Phosphorus Rocket:**
 A similar effect to the smoke rocket, however it creates some fire and has less smoke range.

**Sleeping Smoke Rocket:**
 For some reason we have a smoke subtype that just makes you sleep, so do with that what you will.

**EMP Rocket:**
 This has replaced the ion rifle in every armory, as now we have missiles that make big emp bursts on launch.

**APCBCFSDSGLATM:**
 The "Fuck you" missile, WILL delimb whoever it hits, bounces fucking everywhere, only to be used for dire circumstance (trolling).

## Why It's Good For The Game

This is entirely not good for the game at all, unless it is in which case please refer to the round this somehow didn't cause problems.

## Changelog
:cl:
add: Five new flavors of 84mm rocket
add: A security made rocket launcher to use these (sprites from tgmc)
qol: Tiders are now at severe risk of having a hole blown through their chest with missiles
balance: The ion rifles and dragnet guns are no more, as they have been replaced with rocket launchers
fix: Every single sec balance issue is now fixed
/:cl:

### CODE REVIEW

**Comment by Paxilmaniac** [comment]
apparently this needs to be opened at least once for testmerge, what an awful reality we live in

**Comment by Rowelll** [comment]
also give lethals to sec and make sec armor a riot armor subtype, thank you!!!

**Comment by InsaneRed** [comment]
sec will never be balanced untill we have the option to call in an orbital strike at will

**Comment by necromanceranne** [comment]
Where is my Red Glare full auto rocket launcher. How can I stop crime if I've not levelled the station along with it.

File: code/modules/projectiles/projectile/special/rocket.dm [hunk 1]
```diff
@@ -70,6 +70,77 @@
...
+	explosion(target, flame_range = 7, flash_range = 2)
+
+/obj/projectile/bullet/a84mm/smoke/sleeping
+	name = "\improper RCSD missile"
+	desc = "What's this sleep gas stuff made of anyway?"
+	icon_state = "84mm-sleepsmoke"
+
+/obj/projectile/bullet/a84mm/smoke/sleeping/do_boom(atom/target)
+	playsound(src, 'sound/effects/smoke.ogg', 50, TRUE, -3)
+	var/datum/effect_system/smoke_spread/sleeping/smoke = new
```
Comment (Ghommie) [inline code/modules/projectiles/projectile/special/rocket.dm:139]: That's an instant 20 seconds of sleepy time to everyone who doesn't have a internals of a good gas mask in a five turfs range, mind you.

Comment (Paxilmaniac) [reply code/modules/projectiles/projectile/special/rocket.dm:139]: I am entirely aware of this, which is why they only get three roundstart, at least until they research advanced weapons tech. Even when that is researched, the missiles are fairly expensive to produce.

Comment (Ghommie) [reply code/modules/projectiles/projectile/special/rocket.dm:139]: Ok, just remember there's always teargas if someone asks you to replace these later.

**Comment by Paxilmaniac** [comment]
I'm not making any more changes to this so whatever.

**Pull request closed without merging**